src/pagination/src/interface.ts

```diff
@@ -16,3 +16,12 @@ export type PaginationSizeOption = SelectBaseOption<number, string>
 export type RenderSuffix = RenderPrefix
 export type RenderNext = RenderPrefix
 export type RenderPrev = RenderPrefix
+
+export type RenderItem = (info: {
+  item: {
+    label: string | number
+    type: 'page' | 'fastBackward' | 'fastForward'
+  }
+  showFastBackward: boolean
+  showFastForward: boolean
+}) => VNodeChild
```

Comment:
```
(info: {
  label: string,
  type: 'page' | 'fastBackward' | 'fastForward',
  active: boolean // hover for fastBackward & fastForward, current page for page
}) => VNodeChild
```

Reply:
If type if 'fastBackward' & 'fastForward', active means hovered. If type is 'page', active means it's current page.

Reply:
@jaulz

Reply:
The original showFastBackward & showFastForward is useless for another case and 'page' type.

Reply:
Yep, it's added now. Sorry I missed that before 😊